### ISSUE TITLE
Is there a way to retrieve replies on a tweet?

### ISSUE THREAD

**Issue by paratroid** [Rishikant181/Rettiwt-API#444]
For example if I were to look at [this tweet](https://twitter.com/GergelyOrosz/status/1751691937411662209), is there a way to collect the replies on it? I'm currently using the `tweet.details` method to get the individual tweet, but there doesn't seem to be a way I might be able to chain this together to collect replies on tweets. Any help or advice would be much appreciated! And thank you for all the awesome work you're doing on this API 💯

**Comment by Rishikant181** [comment]
Currently, fetching of replies is not supported, but it's being worked on https://github.com/Rishikant181/Rettiwt-API/issues/433

**Comment by paratroid** [comment]
Absolutely amazing! Thank you! 🥇

**Comment by Rishikant181** [comment]
Duplicate of #433

**Comment by Jeto143** [comment]
Hi @Rishikant181, I was looking to accomplish this for one of my projects, but I noticed you marked #433 as "not planned". Is that not possible? (I was thinking of a `replyTo` field in `TweetFilter`)

Or maybe I missed another way to do it that you've implemented since then?

Thank you!

**Comment by Rishikant181** [comment]
@Jeto143 Fetching replies is proving to be quite the headache owing to the nested nature of the conversations that exist in the replies section of a tweet. I'm still grinding my gears over trying to figure out an elegant and maintainable method to filter out the replies but so far, it has been unfruitful. As such, the issue has been marked as not-planned. As soon as a viable solution is deemed to be somewhat plausible, the issue will be updated.

TL;DR: Fetching and filtering replies is proving to be difficult :(

**Comment by Jeto143** [comment]
@Rishikant181 No worries, thank you for the transparency!

**Comment by Jeto143** [comment]
@Rishikant181 The `conversation_id` operator seems to be the closest, although it only seems to work on opening tweets (not replies). E.g.:

Tweet: https://twitter.com/DaMJFan/status/1451509766866587649
Replies: https://twitter.com/search?q=conversation_id%3A1140437409710116865

(I'm assuming Rettiwt-Core performs these types of query from the looks of [its code](https://github.com/Rishikant181/Rettiwt-Core/blob/dev/src/models/args/TweetFilter.ts#L154), but I'm not 100% sure.)
